fatrace.8

```diff
@@ -192,6 +192,11 @@ Show only events for this command.
 .B \-j\fR, \fB\-\-json
 Write events in JSONL format (see definition above).
 
+.TP
+.B \-i\fR, \fB\-\-inclusive
+Include possible false negatives, i.e. events where missing data makes filtering ambiguous.
+If \fB\-\-command\fR is specified, events with unknown command will be printed.
+
```

Comment:
IMHO this is unnecessary complexity. Your reasoning makes sense, let's just do that as default.

Reply:
Agreed. Opened #53.